### PR TITLE
Add suggested Usage/API

### PR DESCRIPTION
**Pull request by jonescc**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,21 @@
-# netcdf-subset-service
-NetCDF Subset Service
+# NetCDF Subset Service
+
+## Usage
+
+### 
+
+    http://localhost:8080/netcdf-subset?typeName=${LAYER_NAME}&CQL_FILTER=INTERSECTS(${GEOMETRY_ATTR},${GEOMETRY_LIT} AND juld >= '${FROM_TIMESTAMP}' AND juld <= '${TO_TIMESTAMP}'
```
Comment (danfruehauf) [inline README.md]: I think you didn't close the `(` after `INTERSECTS`.

Comment (jonescc) [inline README.md]: And I need to parameterize the timestamp  field

Comment (danfruehauf) [inline README.md]: So... wait for that?

**Comment by jonescc** [comment]
@danfruehauf, comments addressed

**Comment by danfruehauf** [comment]
Fine with me, just run it through @pblain as well so we know we meet the requirements 100%.

**Comment by pblain** [comment]
@jonescc , looks good.

**Pull request merged**